swift-lang/src/main/resources/ecocode_swift_profile.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "ecoCode",
   "ruleKeys": [
-    "EIDL001",
-    "EIDL002",
-    "EPOW001",
-    "EPOW002",
-    "ESOB001",
-    "ESOB002",
-    "ESOB003",
-    "ESOB005",
-    "ESOB006",
+    "EC14",
+    "EC15",
+    "EC16",
+    "EC17",
+    "EC18",
+    "EC19",
+    "EC20",
+    "EC21",
+    "EC23",
     "ESOB007"
```

Comment:
why is ESOB007 remaining ?

Reply:
Because this rule was created while the common rule repo was initiated.
It should me moved to the common repo later.
Also: EC21 as no EC5XX equivalent

Reply:
maybe create a new EC5xx for EC23 to be homogeneous ... ?